rasa/core/policies/mapping_policy.py

```diff
@@ -32,6 +37,28 @@ def __init__(self, priority: int = 3) -> None:
 
         super(MappingPolicy, self).__init__(priority=priority)
 
+    @classmethod
+    def validate_against_domain(
+        cls, ensemble: Optional["PolicyEnsemble"], domain: Optional[Domain]
+    ):
+        if not domain:
+            return
+
+        has_mapping_policy = ensemble is not None and any(
+            isinstance(p, cls) for p in ensemble.policies
+        )
+        has_triggers_in_domain = any(
+            [
+                "triggers" in properties
+                for intent, properties in domain.intent_properties.items()
+            ]
+        )
+        if has_triggers_in_domain and not has_mapping_policy:
+            raise InvalidDomain(
+                "You have defined triggers in your domain, but haven't "
+                "added the MappingPolicy to your policy ensemble."
```

Comment:
maybe add link to the docs or a suggestion how to solve it (`please add the MappingPolicy to your policy config or remove the triggers from the domain`)